src/ServiceControl.Audit/Auditing/MessagesView/GetMessages2Controller.cs

```diff
@@ -0,0 +1,56 @@
+namespace ServiceControl.Audit.Auditing.MessagesView;
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Infrastructure;
+using Infrastructure.WebApi;
+using Microsoft.AspNetCore.Mvc;
+using Persistence;
+
+[ApiController]
+[Route("api")]
+public class GetMessages2Controller(IAuditDataStore dataStore) : ControllerBase
+{
+    [Route("messages2")]
```

Comment:
Maybe better as `messages-v2` or `messages/v2`?

Reply:
Yeah, hard to name things